Casks/font-437-plus.rb

```diff
@@ -13,6 +13,11 @@
   desc "Collection of classic text mode fonts from DOS-era IBM PCs and compatibles"
   homepage "https://int10h.org/oldschool-pc-fonts/"
 
+  livecheck do
+    url "https://int10h.org/oldschool-pc-fonts/download/"
+    regex(/oldschool_pc_font_pack_v(\d\.\d)_win\.zip/i)
```

Comment:
Thanks @Kreeblah - this changes aligns the regex with the [livecheck regex guidelines](https://docs.brew.sh/Brew-Livecheck#regex-guidelines).

Additionally, please submit PRs from a branch in your repo rather than your `master` branch. This allows maintainers to push additional commits to your PR.

```suggestion
    regex(/href=.*?oldschool[._-]pc[._-]font[._-]pack[._-]v?(\d+(?:\.\d+)+)[._-]win\.zip/i)
```

Reply:
I didn't realize you couldn't modify PRs from the `master` branch.  I'll keep that in mind.

Same with the livecheck regex guidelines.  I didn't realize those existed, either, but I'll keep them in mind for the future.

Reply:
@Kreeblah No worries at all. Thanks for your contribution to homebrew!